packages/dashboard-core-plugins/src/panels/FileExplorerPanel.tsx

```diff
@@ -169,8 +169,12 @@ export class FileExplorerPanel extends React.Component<
     });
   }
 
-  getPath(file: FileStorageItem): void {
-    const path = FileUtils.makePath(file.filename);
+  handleSelectionChange(selectedItems: FileStorageItem[]): void {
+    let path = '/';
+    if (selectedItems.length === 1 && selectedItems[0].type === 'directory') {
```

Comment:
I don't think you need to bother checking the type here - assume you want to create a directory in the same directory that is selected, whether that is a dir or a file.